public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -131,8 +131,7 @@ export function useAllAlertmanagerAbilities(): Abilities<AlertmanagerAction> {
     // only Grafana flavored alertmanager supports exporting
     [AlertmanagerAction.ExportContactPoint]: [
       isGrafanaFlavoredAlertmanager,
-      ctx.hasPermission(notificationsPermissions.provisioning.read) ||
-        ctx.hasPermission(notificationsPermissions.provisioning.readSecrets),
+      ctx.hasPermission(notificationsPermissions.read),
```

Comment:
why do we remove the other permissions?. In the BE [PR](https://github.com/grafana/grafana/pull/76082/files#diff-627a793303894bbf1b53347d4c9c0316e28159fa392b13786f80553dc71e8dd4R194) it seems that export contact point will be allowed if any of these permissions is present.
I think we should only add and OR with this permission.

Reply:
Good question! I wasn't sure about this change, but if you don't have `notificaitonsPermissions.read` then you cannot see the notification policies and contact point pages at all. 
`provisioning.readSecrets` doesn't give you permission to see any page, so I removed it

Reply:
yes, it makes sense ! but maybe we should update the api permissions to be aligned with the UI?

Reply:
They don't have to be aligned in this case. You can you provisioning API without using the UI
@yuri-tceretian wdyt?

Reply:
If user with only provisioning permissions cannot see the UI then I think this is ok to only allow `notificationsPermissions.read`. 
I do not think we should change the backend code, though. 
1. There is a possibility that users could use it in automation. 
2. This is provisioning endpoint and therefore it is logically correct to accept provisioning permissions.

Reply:
It is worth mentioning that we should check `readSecrets` downstream to let the user return decrypted integration. Ideally, I would like to show a toggle that would be disabled by default and that would control whether return secrets decrypted or redacted.

Reply:
From what I see, `readSecrets` is used by the backend, and if the user has this permission, they'll get a decrypted version. Otherwise, it will be redacted.
We could add the switch, but we'd need to pass this setting to the API. Is that supported?

Reply:
IIRC it is supported in the API with `decrypt=true|false`

Reply:
I don't want to introduce more UI changes in this PR, because we want to backport it to 10.2
I created an issue to add a `decrypt=true|false` toggle to do it in a separate PR